tests/checking-pr-readiness/fixtures/run-assessment-checks.py

```
@@ -626,44 +623,134 @@ def resolve_live_repository(repo: Path) -> tuple[str | None, list[str]]:
 
 
 def reviewer_configuration(repo: Path, revision: str) -> list[dict[str, Any]]:
-    _, content = evidence_blob(repo, revision, ".github/automated-reviewers.json")
+    _, content = evidence_blob(repo, revision, REVIEWER_CONFIG_PATH)
     document = json.loads(content)
     reviewers = document.get("automated_reviewers")
     if not isinstance(reviewers, list):
         raise FixtureError("repository automated-reviewer discovery is invalid")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)